src/main/java/org/jenkins_ci/plugins/flexible_publish/FlexiblePublisher.java

```diff
@@ -328,7 +328,7 @@ public static String getBuildStepShortName(List<BuildStep> buildStepList) {
                         new Function<BuildStep, String>() {
                             @Override
                             public String apply(BuildStep input) {
-                                return getBuildStepShortName(input);
+                                return (input != null) ? getBuildStepShortName(input) : "";
```

Comment:
<BugInstance type='NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE' priority='Normal' category='STYLE' message='input must be non-null but is marked as nullable' lineNumber='331'/>

input is nullable.